README.md

```diff
@@ -0,0 +1,37 @@
+# Ubuntu image builder
+
+[![Build & Tests Status](https://travis-ci.org/canonical/ubuntu-image.svg?branch=master)](https://travis-ci.org/canonical/ubuntu-image)
+[![codecov](https://codecov.io/gh/canonical/ubuntu-image/branch/master/graph/badge.svg)](https://codecov.io/gh/canonical/ubuntu-image)
+[![Go Report Card](https://goreportcard.com/badge/github.com/canonical/ubuntu-image)](https://goreportcard.com/report/github.com/canonical/ubuntu-image)
+
+
+This tool is used to build Ubuntu images.  Currently it only builds Snappy
+images from a model assertion, but it will be generalized to build more
+(eventually all) Ubuntu images.
+
+
+# Requirements
+
+Ubuntu 16.04 (Xenial Xerus) is the minimum platform requirement, but Ubuntu
```

Comment:
Since this is a 'new start', I think it will be wise to start off with the base minimum requirement being a series that is still fully supported. Xenial is now basically in ESM mode, so let's maybe mark bionic as the minimum requirement?